src/WindowsTracing/KrabsTracer.cpp

```diff
@@ -57,12 +98,13 @@ void KrabsTracer::Stop() {
   trace_thread_->join();
   trace_thread_ = nullptr;
   OutputStats();
+  EnableSystemProfilePrivilege(false);
   context_switch_manager_ = nullptr;
 }
 
 void KrabsTracer::Run() {
   orbit_base::SetCurrentThreadName("KrabsTracer::Run");
-  trace_.start();
+  trace_.process();
```

Comment:
What is the difference between `start()` and `process()`? Is `start()` no longer needed?

Reply:
start() internally calls open() and process(), which is blocking. open() creates a session handle which we need in order to call `set_trace_information`. We need to use open()/process() instead of start() so that we can access the session handle before the blocking call to process(). See [comment in krabsetw PR](https://github.com/microsoft/krabsetw/pull/181#discussion_r714400551).

Reply:
Thanks!